### PR TITLE
Added category: Adjective

### PR DESCRIPTION
**Pull request by darensin01**

### CODE REVIEW

File: test/categories/test_adjective.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.adjective import Adjective
+import random
+
+
+class TestAdjectives(unittest.TestCase):
+  def setUp(self):
+    self.adjective = Adjective()
+
+  def test__get_values__returns_list(self):
```
Comment (gabrielbarker) [inline test/categories/test_adjective.py:10]: The way unit tests for categories are written in this repo has changed recently. If you pull the latest changes and take a look at another test class, you'll see that it's been simplified a little

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
Hey @darensin01 , thanks a lot for the contribution! I just have one small change if you don't mind. I've included a comment below. Thanks!

**Review by darensin01 — COMMENTED** [review]
Modified test_adjective

**Review by gabrielbarker — APPROVED** [review]
Nice! I really like this category

**Comment by gabrielbarker** [comment]
Thanks a lot for the contribution @darensin01 !

**Comment by darensin01** [comment]
Thanks! :)

**Pull request merged**